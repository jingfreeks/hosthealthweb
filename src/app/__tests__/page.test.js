import { render } from "@testing-library/react";
import Page from "../page";

describe("Page", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
