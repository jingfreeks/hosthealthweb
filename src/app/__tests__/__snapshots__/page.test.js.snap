// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders homepage unchanged 1`] = `
<div>
  <main
    class="className"
  />
</div>
`;
